get-phing.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+wget http://www.phing.info/get/phing-latest.phar -O phing.phar
```

Comment:
@nblair oh yeah, duh

I've been considering whether Phing should be a developer requirement, much like we require people to have mvn installed.